src/components/app.js

```diff
@@ -46,12 +46,14 @@ class App extends React.Component {
         await this.props.loadUserData(uid, this.showErrorPage);
         this.setState({ checkedAuth: true });
       } else {
-        this.setState({ checkedAuth: true, errorMsg: "No UID provided with user" });
+        this.props.loadFailure();
+        this.setState({ checkedAuth: true });
       }
     } else {
       console.log("no user found");
       this.props.clearUserData();
-      this.setState({ checkedAuth: true, errorMsg: "" });
+      this.props.loadFailure("No user could be found");
```

Comment:
This else isn't a bad thing, it just means that no one is logged into firebase yet. So no need to create an error here!